public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -469,20 +576,6 @@ $(function () {
                 e.stopPropagation();
             }
         }
-        // Check that the font-awesome icon looks right
```

Comment:
will the schema validation do that, when somebody uploads their schema with a malformed FA icon?

Reply:
Currently there is no validation so if someone uploads something weird then it will, yeah, probably break.

Reply:
Ok, I added a validation step when building the schema rows. Now if there's something weird it will just delete the key from the schema and treat it as if there was no icon. It throws an error message to the browser console.